### PR TITLE
Dhdl bugfixes python3

### PR DESCRIPTION
**Pull request by zetadin**
Some bug fixes for analyzing dHdl files. Especially useful when a lot of the non-equilibrium simulations crash producing corrupt dHdl files. Also added a unit test for the relevant analysis script.

### CODE REVIEW

File: src/pmx/scripts/analyze_dhdl.py [hunk 1]
```diff
@@ -608,6 +608,9 @@ def main(args):
                                                                                  p=prec, u=units), quiet=quiet)
 
     _tee(out, ' ========================================================', quiet=quiet)
+    
+    #close the outputfile, forsing a fush to disk, so that the unit test can read it properly
+    out.close();
```
Comment (vgapsys) [inline src/pmx/scripts/analyze_dhdl.py]: semicolon

File: tests/test_dhdl.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                s=line.split()
+                print(s)
+                bar=float(s[-2])
+            elif("BAR: Std Err (analytical) =" in line):
+                s=line.split()
+                print(s)
+                barerr=float(s[-2])
+    
+    assert_almost_equal(bar, dG*0.98, decimal=2);
+    assert_almost_equal(barerr, 0.17, decimal=2);
```
Comment (vgapsys) [inline tests/test_dhdl.py]: more semicolons

File: src/pmx/scripts/analyze_dhdl.py (hunk 1 above)
Comment (zetadin) [reply src/pmx/scripts/analyze_dhdl.py]: Any reason to add semicolons to just those lines?
I was going off of the [google style guide for python on semicolons](http://google.github.io/styleguide/pyguide.html#31-semicolons).

Comment (vgapsys) [reply src/pmx/scripts/analyze_dhdl.py]: ah, right, it is a _removed_ semicolon. That is what I actually wanted - to remove them :)

Comment (zetadin) [reply src/pmx/scripts/analyze_dhdl.py]: After looking at the code, I see I left semicolons all over the place

File: tests/test_dhdl.py (hunk 2 above)
Comment (zetadin) [reply tests/test_dhdl.py]: removed

**Pull request merged**